substrate/client/authority-discovery/src/worker.rs

```diff
@@ -753,6 +891,7 @@ fn sign_record_with_authority_ids(
 	peer_signature: Option<schema::PeerSignature>,
 	key_store: &dyn Keystore,
 	keys: Vec<AuthorityId>,
+	include_creation_time: bool,
```

Comment:
Is there a reason to not assume it's always `true`?

Reply:
It is false in some of the tests, to check we accept records without creation time.